mobile/examples/object_detection/ios/ORTObjectDetection/download.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# ============================================================
+
+MODELS_URL="https://tfhub.dev/tensorflow/lite-model/ssd_mobilenet_v1/1/metadata/1?lite-format=tflite"
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+cd "$SCRIPT_DIR"
```

Comment:
is it necessary to `cd`?

Reply:
Right. This can be omitted. Also will delete the `ModelsAndData` dir later as the labels data and model can all be generated by the script.